### PR TITLE
Theme color

### PR DESCRIPTION
**Pull request by bbondy**
This adds an event to webviews for when meta tags with a theme-color are encountered.

### CODE REVIEW

File: atom/browser/api/atom_api_web_contents.cc [hunk 1]
```diff
@@ -489,6 +489,16 @@ void WebContents::MediaPaused() {
   Emit("media-paused");
 }
 
+void WebContents::DidChangeThemeColor(SkColor theme_color) {
+  char themeColor[8] = { 0 };
+  snprintf(themeColor, sizeof(themeColor),
```
Comment (zcbenz) [inline atom/browser/api/atom_api_web_contents.cc]: `base::StringPrintf` is preferred for memory safety.

File: docs/api/web-contents.md [hunk 2]
```diff
@@ -221,14 +221,6 @@ Emitted when `webContents` wants to do basic auth.
 
 The usage is the same with [the `login` event of `app`](app.md#event-login).
 
-### Event: 'media-started-playing'
```
Comment (zcbenz) [inline docs/api/web-contents.md]: I think these events should be kept in `WebContents`, and `did-change-theme-color` can also be added here.

File: docs/api/web-view-tag.md [hunk 3]
```diff
@@ -634,3 +634,17 @@ Fired when a plugin process is crashed.
...
+
+### Event: 'media-paused'
+
+Emitted when media is paused or done playing.
+
+### Event: 'did-change-theme-color'
+
+Emitted when a page's theme color changes. This is usually due to encountering a meta tag:
+
+    <meta name='theme-color' content='#ff0000'>
```
Comment (zcbenz) [inline docs/api/web-view-tag.md]: We use following way of pasting code so code highlight can work:

<pre>
```html
your code here
```
</pre>

**Comment by bbondy** [comment]
Thanks for the guidance, I'll make it so.

**Comment by bbondy** [comment]
@zcbenz Updated. 
Please have another look, thanks.

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**